cdap-watchdog/src/main/java/io/cdap/cdap/metrics/query/MetricsQueryHelper.java

```diff
@@ -109,6 +109,10 @@
 
       .put(Constants.Metrics.Tag.PROVISIONER, "provisioner")
 
+      .put(Constants.Metrics.Tag.STATUS_CODE, "statuscode")
```

Comment:
Why add here? Custome metrics context (e.g. the one emitted by program or extension) shouldn't need to be defined here. The one who query simply provide "sc", "reg" as the context.

Reply:
removed.